src/main/java/edu/hw1/Task1.java

```diff
@@ -0,0 +1,43 @@
+package edu.hw1;
+
+import java.io.IOException;
+
+public class Task1 {
+    public final String uncorrectDigitInInput = "некоректные цифры во вводе";
+    public final String uncorrectInput = "некоректный ввод";
```

Comment:
`BUG` ```suggestion
    public final String incorrectInput = "некоректный ввод";
```
Кажется правильно incorrect